### DISCUSSION TITLE
PostgreSQL 14.11 is not able to identify the postmaster daemon.

### DISCUSSION THREAD

**Discussion by vigneshkumar2016** [docker-library/postgres#1209] [General]
PostgreSQL 14.11 runs out on error.
Image: Official Image
Version: 14.11
OS: Debian
OS version: bullseye
Container: yes

PostgreSQL errors out on 14.11
Segmentation fault (core dumped). this is quite strange.

PostgreSQL 14.5, 14.6, 14.0 works just fine.

Seems like the compiler is overshooting on the address space to load postgres.

any help here is much appreciated.

**Comment by ImreSamu** [comment]
Hi @vigneshkumar2016 ,

It's possible that the issue you're encountering is related to `libseccomp`.
If that's the case, you might consider updating both `libseccomp` and docker on the host that's running the containers.
( or just use `postgres:14.11-bullseye` )

For detailed information, including the root cause and potential workarounds, please visit:
https://github.com/docker-library/postgres/issues/1100#issuecomment-1599660628

**Comment by vigneshkumar2016** [comment]
> libseccomp

Thanks, this works for me. However the traceback results and coredump looks similar with that of the one which is working and failing. pretty strange to debug. But clearly getting libraries are not getting loaded to the container memory.

**Comment by tianon** [comment]
Yeah, the seccomp blocks happen at the kernel level, so strace is one of the few tools that shows what's happening and even then it's usually an obscure `EPERM` in places it shouldn't be and no real indicator that it's actually seccomp blocking it. :sob:

**Reply by vigneshkumar2016** [reply]
Yeah seems like it has to read the XFR register call. but failing to do so.. its not able to identify what to load to kernel. either fails with Segmentation fault. possibly expecting a fix in generic.
